ztypes_netbsd_32bit_int.go

```diff
@@ -0,0 +1,13 @@
+// +build openbsd
+// +build 386 amd64 arm
```

Comment:
The file says 32bit, yet we have amd64 here, is that expected? Some OS/Archs do have the same 32bit struct indeed, just making sure.

Reply:
Just copied the file name from openbsd.  The types are the same on all architectures netbsd supports.